mobile/src/common/enums/course/course-description-first-words-count.enum.ts

```diff
@@ -0,0 +1,5 @@
+enum CourseDescriptionFirstWordsCount {
+  DEFAULT_COUNT = 40,
+}
+
```

Comment:
It's just a constant and we won't reuse it, so move it to components/course/common/constants/constants.ts. Also use length not by words

Reply:
OK, but I only wanted not to cut the words

Reply:
It's ok that we will cut it by length, this way we always will have similar height